LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/authentication/KeyVaultAuthenticationClient.java

```diff
@@ -37,9 +37,9 @@
  */
 public class KeyVaultAuthenticationClient implements IAccessTokenSupplier {
 
-    private final static String CLIENT_ID = "4bc6e96f-bd23-408f-8ecb-a7a7145463f9";
-    private final static String SCOPE = "https://vault.azure.net/.default";
-    private final static String CERTIFICATE_ALIAS = "AutomationRunner";
+    private final static String CLIENT_ID = "f62c5ae3-bf3a-4af5-afa8-a68b800396e9";
+    private final static String SCOPE = "https://msidlab.com/.default";
+    private final static String CERTIFICATE_ALIAS = "LabVaultAccessCert";
```

Comment:
Question, if this cert is meant to replace our usage of a lab secret, what was the old cert being used for?

Reply:
Old one was for the AutomationRunner app, which we are not using anymore.